### DISCUSSION TITLE
Add Roblox in the Block/Unblock Services list

### DISCUSSION THREAD

**Discussion by muhepd** [AdguardTeam/AdGuardHome#3083] [Ideas]
Roblox is a massive gaming service nowadays, specially for young kids/teens, I see that Steam/EpicGames are there, I believe Roblox should be there as well.

I know I can add Roblox in the custom rules, but it is easier to manage if it is just a switch in the Block/Unblock Services list.

Thanks.
![Roblox_Service](https://user-images.githubusercontent.com/53619632/117190631-aee32000-adad-11eb-8981-65ad8b0c8e2a.PNG)

**Comment by 522122** [comment]
Nothing? :D

**Reply by Dynamic5912** [reply]
It's already there? And has been for some time now
